### DISCUSSION TITLE
Issue with Macros

### DISCUSSION THREAD

**Discussion by nick-softwaredev** [pointfreeco/swift-composable-architecture#2760] [Q&A]
Hi! 

I am trying to implement a composable architecture(1.7.2) in my app, but for some reason its macros are not recognized by Xcode compiler. 

What i've tried: https://forums.swift.org/t/ignore-macro-validation-using-xcodebuild-command/68125 but no luck yet. 

<img width="936" alt="Screenshot 2024-02-01 at 7 25 40 PM" src="https://github.com/pointfreeco/swift-composable-architecture/assets/48436203/3e6ad5f5-5076-450d-b149-1adf5429bf6c">

XCode 15.2(15C500b)

Thanks in advance.

**Comment by mbrandonw** [comment]
Hi @nick-softwaredev, please provide more information (Xcode version, TCA version), or ideally a sample project that reproduces the problem.

**Reply by nick-softwaredev** [reply]
Just updated @mbrandonw

**Reply by mbrandonw** [reply]
If you are using the newest Xcode and TCA then you will need to provide a project that demonstrates the problem. Macros certainly do work, so there must be something happening with your local project.

**Reply by nick-softwaredev** [reply]
You right, my bad! Just tried to replicate with brand new sample project and everything works fine.  But while adding into existing project,  it is not recognized, any advice on where to look for potential issue ?  

Thank you! @mbrandonw

**Reply by mbrandonw** [reply]
It's hard to say without a project to look at. Perhaps try cleaning the project and deleting derived data.
